modules/homefinder/widgets/class-homes-for-sale-widget.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * HomeFinder Homes for Sale Widget (http://www.homefinder.com/widgets/)
+ *
+ * @package RE-PRO
+ */
+
+	/* Exit if accessed directly. */
+if ( ! defined( 'ABSPATH' ) ) { exit; }
+
+/**
+ * HomeFinderHomesForSale class.
+ *
+ * @extends WP_Widget
+ */
+class HomeFinderHomesForSale extends WP_Widget {
+
+	/**
+	 * __construct function.
+	 *
+	 * @access public
+	 * @return void
+	 */
+	public function __construct() {
+
+		parent::__construct(
+			'homefinder_homes_for_sale',
+			__( 'HomeFinder Homes for Sale', 're-pro' ),
+			array(
+				'description' => __( 'Display a search box from HomeFinder.com', 're-pro' ),
+				'classname'   => 're-pro re-pro-widget homefinder-widget homefinder-homes-for-sale',
+				'customize_selective_refresh' => true,
+			)
+		);
+	}
+
+	/**
+	 * Widget function.
+	 *
+	 * @access public
+	 * @param mixed $args Arguments.
+	 * @param mixed $instance Instance.
+	 */
+	public function widget( $args, $instance ) {
+
+		$widget_id = ! empty( $args['widget_id'] ) ? $args['widget_id'] : '';
+		$title = ! empty( $instance['title'] ) ? $instance['title'] : '';
+
+		echo $args['before_widget'];
+
+		echo $args['before_title'] . esc_attr( $title ) . $args['after_title'];
+
+		$homefinder_widgets = new HomeFinderWidgets();
+
+		$homefinder_widgets->get_homes_for_sale_widget();
+
+		echo $args['after_widget'];
+	}
+
+	/**
+	 * Form function.
+	 *
+	 * @access public
+	 * @param mixed $instance Instance.
+	 * @return void
+	 */
+	public function form( $instance ) {
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Expected 0 spaces between opening bracket and argument "$instance"; 1 found
* Expected 0 spaces between argument "$instance" and closing bracket; 1 found
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->